docs/zh_cn/04-developer-guide/do_regression_test.md

```diff
@@ -177,21 +194,27 @@ models:
     pipelines:
       - *pipeline_ort_detection_dynamic_fp32
       - *pipeline_trt_detection_dynamic_fp16
+
+      # 特殊的 pipeline 可以这样加入
+      - convert_image: xxx
+        backend_test: xxx
+        sdk_config: xxx
+        deploy_config: configs/mmocr/text-detection/xxx
 ```
 
-## 3. 生成的报告
+## 4. 生成的报告
 
 ### 模板
 
-|| model_name | model_config | task_name       | model_checkpoint_name | dataset  | backend_name | deploy_config | static_or_dynamic | precision_type | conversion_result | fps | metric_1 | metric_2 | metric_n | test_pass |
+|| Model | Model Config | Task       | Checkpoint | Dataset  | Backend | Deploy Config | Static or Dynamic | Precision Type | Conversion Result | FPS | metric_1 | metric_2 | metric_n | Test Pass |
```

Comment:
|| Task | Model  | Model Config       | Checkpoint | Dataset  | Backend | Deploy Config | Static or Dynamic | Precision Type | Conversion Result | FPS | metric_1 | metric_2 | metric_n | Test Pass? |
|------------|--------------|-----------------|-----------------------|----------|--------------|---------------|-------------------|----------------|-------------------|---|----------|----------|-----------|-----------|-----|
| 序号         | 任务名称         | 模型名称 | 模型配置文件路径     | `.pth`模型路径 | 数据集名称        | 后端名称    |  部署配置文件路径    | 动态 or 静态          | 计算精度           | 模型转换结果       | FPS 数值     | 指标 1 数值 | 指标 2 数值        | 指标 n 数值         |  测试用例是否通过  |

Reply:
I think 'FPS' is not necessary, because `regression_test.py` focuses on verifying whether deployment model metrics are the same as the training models.

Reply:
I think we can keep it.
Cuz I think we also need to keep attention on FPS when test different backend, such as fp16 will faster than fp32.